game/script-topics.rpy

```diff
@@ -1044,61 +1044,238 @@ init 5 python:
+    extend 4csqfllsbl " Alright?"
+    n 2clrflsbr "I'm being serious here."
+    n 2cnmslsbr "Call it quits on whatever you were doing and just come back to it later.{w=0.75}{nw}"
+    extend 1csqbosbr " Got it?"
+    n 3ccstr "Work or some dusty old assignment can wait if you're just gonna make yourself even worse trying to impress someone."
+    n 3csqbo "No,{w=0.2} your boss or professor isn't going to drop dead if some stinky report isn't done today."
+    n 1ccsss "Heh.{w=0.75}{nw}"
+    extend 4clrss " Anyway,{w=0.2} think about it."
+    n 3tnmbo "It isn't like you're going to pull some kind of miracle if you try and push through it all."
+    n 3ccsaj "All you're gonna achieve is make yourself feel like crap for even longer.{w=0.75}{nw}"
+    extend 3fcsca " It's totally pointless."
+    
+    if Natsuki.isEnamored(higher=True):
+        n 2cllca "...And besides,{w=0.2} [player]."
+        n 2cllpulsbl "You know I'm not going to get mad or anything if you have to take a rain check on our time together..."
+        n 4knmbolsbl "Right?"
+        n 4klrbolsbl "..."
+        n 2ccsfllesisbl "..."
+        n 2cnmsll "...Look."
+        n 1fcspul "I...{w=1}{nw}"
+        
         if Natsuki.isLove(higher=True):
-            $ chosen_endearment = jn_utils.getRandomEndearment()
-            n 4kwmsml "I love you,{w=0.1} [chosen_endearment]."
-            n 1kwmnvl "..."
-            return
+            extend 1ncspul " really...{w=1}{nw}"
+        
+        $ chosen_descriptor = jn_utils.getRandomEndearment() if Natsuki.isLove(higher=True) else player
+        extend 4csrbol " care about you,{w=0.2} [chosen_descriptor].{w=0.75}{nw}"
+        extend 5cnmbol " You should seriously know that by now."
+        n 3ccsajl "...Just like you should know you I'm {i}not{/i} gonna be impressed by some weird macho display of toughing it out."
+        n 3cnmpol "Capiche?"
+        n 3cslbol "..."
+
+    elif Natsuki.isAffectionate(higher=True):
+        n 2cllflsbr "A-{w=0.2}and besides,{w=0.2} [player].{w=0.75}{nw}"
+        extend 2ccssllsbl " I'm not {i}that{/i} selfish."
+        n 4csrajlsbl "You {i}do{/i} know I won't get mad or anything if you really have to go for a while..."
+        n 4cnmsllsbl "Right?"
+        n 1cslbolsbl "..."
+
+    else:
+        n 2fcstr "A-{w=0.2}and besides,{w=0.2} [player]."
+        n 2clrajsbl "It's not like I'm gonna get mad at you or anything like that either if you gotta duck out.{w=0.75}{nw}"
+        extend 2clrsll " Even I'm not that much of a jerk."
+        n 1clrbol "..."
+
+    n 4ccsflsbr "A-{w=0.2}anyway."
+    n 4ullaj "I've gone on way long enough already,{w=0.5}{nw}"
+    extend 2cnmca " so I'm just gonna say this,{w=0.2} [player]."
+    n 2fcsaj "You might end up with a crappy back or nasty eyes if you aren't careful..."
+    n 3fcspo "...But that's gonna be nothing compared to the grade-A earache you're getting if you come crying to me later!"
```

Comment:
With the different `arms_crossed` poses it looks like she is clipping through the desk here.

Reply:
Good catch, I'll adjust